pkg/oci/load_balancer_security_lists.go

```diff
@@ -362,6 +378,23 @@ func getNodeIngressRules(
 	ingressRules := []core.IngressSecurityRule{}
 
 	for _, rule := range rules {
+		// Remove (do not re-add) any rule that represents the old case when
+		// mutating a single ranged backend port or health check port.
+		if rule.TcpOptions != nil && rule.TcpOptions.DestinationPortRange != nil &&
```

Comment:
This part of the fix. I wanted to be on the safe side so made it very restrictive. It should only target the 'single port rule mutation' described.